gulpfile.js

```diff
@@ -352,46 +352,3 @@ gulp.task("dev", () => {
 		},
 	);
 });
-gulp.task("installDevDeps", function(done) {
-	var deps = [
-		"babel-core",
-		"babel-generator",
-		"babel-preset-latest",
-		"babel-plugin-transform-es2015-modules-commonjs",
-		"babylon",
-		"cheerio",
-		"@angular/core",
-		"stylus",
-		"less",
-		"postcss",
-		"postcss-selector-parser",
-		"marked",
-		"node-sass",
-		"uglify-js",
-		"uglify-es",
-		"source-map@0.5.7",
```

Comment:
Not sure if it's needed here.

Reply:
I've updated it and tested. There is a 1.0.0-rc.2 release, I hope it will stabilise soon.

Reply:
Actually, we don't have tests for it... So you can remove it

Reply:
It's a dependency, not a devDependency. Are you sure? I'm fairly certain it should stay as is.

Reply:
oh yeah you are right. it's just not used. It's for those who don't use any transpilers.